test/azure/legacy/Expected/AcceptanceTests/PackageModeBatch/azure/packagemode/batch/v0/_client.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import Any, TYPE_CHECKING
+
+from azure.core import PipelineClient
+from azure.core.rest import HttpRequest, HttpResponse
+
+from ._configuration import BatchV0ClientConfiguration
+from ._serialization import Deserializer, Serializer
+from .operations import HttpSuccessOperations
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Dict
+
+
+class BatchV0Client:  # pylint: disable=client-accepts-api-version-keyword
```

Comment:
nit: doing v0 and v1 as your two clients makes this seem like a multiapi package. I think it'd be clearer if you do two something more like purview, where you have the package names with `-`s in them and where the clients are actually different and not diff vresions of the same client

Reply:
Got it and I will change the client name